tests/integration_tests/lightgbm_tuner_tests/test_optimize.py

```diff
@@ -20,6 +23,8 @@
 from optuna.integration._lightgbm_tuner.optimize import LightGBMTuner
 from optuna.integration._lightgbm_tuner.optimize import LightGBMTunerCV
 import optuna.integration.lightgbm as lgb
+from optuna.samplers import TPESampler
+import optuna.study as study
```

Comment:
How about removing this line and replace `study.create_study` with `optuna.create_study` in both `test_tune_best_score_reproducibility` functions for consistency?

Reply:
done